azure-storage-cpp-lite/src/storage_url.cpp

```diff
@@ -1,7 +1,27 @@
+#include <ctime>
+
 #include "storage_url.h"
 
-namespace microsoft_azure {
-    namespace storage {
+#include "constants.h"
+
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
```

Comment:
There are lot of changes in this file, is it just taken from the storage-cpp-lite code directly ?

Reply:
code structure is a different so I did not and cannot pull it directly. However, I matched all the encodings carefully with their latest and it works. Even the client tested it. It passed the private preview.